tensorboard/summary/writer/BUILD

```diff
@@ -56,6 +56,20 @@ py_test(
     ],
 )
 
+py_test(
+    name = "event_file_writer_fsspec_test",
+    size = "small",
+    srcs = ["event_file_writer_fsspec_test.py"],
+    main = "event_file_writer_fsspec_test.py",
+    srcs_version = "PY3",
+    tags = ["support_notf"],
+    deps = [
+        ":writer",
+        "//tensorboard:test",
+    ],
+)
+
+
```

Comment:
Could you remove the extra newline to satisfy buildifier?

Reply:
done